### ISSUE TITLE
Broken parsing of triple-quoted strings in arrays

### ISSUE THREAD

**Issue by bugreport1234** [uiri/toml#123]
I've got a toml file like this:
```
x=["""hello
world"""]
```
This looks like valid toml to me. However parsing it with the library gives the result:
```
>>> with open("test.toml") as file:
...     x = toml.load(file)
...
>>> x
{'x': ['h']}
```
It seems the string is just completely cut off after the first character. Changing the string content sometimes results in other cutoffs:
```
x=["""this is a test
"""]
```
results in:
```
{'x': ['this is a ']}
```

I'm using Python 3.6.1 and toml github master on windows.

**Comment by uiri** [comment]
Hi,

Thank you for the bug report. I think this is a duplicate of #113. Do you still get this issue with Python on Windows if you use Unix (`\n`) rather than Windows/DOS (`\r\n`) linebreaks?

**Comment by bugreport1234** [comment]
Yes, replacing `\r\n` with `\n`  (or even just `\r`) doesn't change the output, the string is still cut off.

**Comment by bugreport1234** [comment]
I also just tested this on linux, and get the same bug, the string is cut off as well.

**Comment by uiri** [comment]
Cool. I've verified this bug. Arrays don't seem to like multiline strings.

**Comment by gaborbernat** [comment]
@uiri any plan to fix this?

**Comment by rudy-6-4** [comment]
Could be linked to #204, but the fix '#293' for #204 will not work on this case.
